src/lib/model/character.ts

```diff
@@ -2,7 +2,13 @@ import { characters } from './statics/characters'
 import { writable, type Writable } from 'svelte/store'
 import unnamed from '/static/illustrations/characters/unnamed.webp'
 import type { Task } from './tasks'
-const pictures = import.meta.glob('/static/illustrations/characters/*.webp')
+const pictures = import.meta.glob(
+	'/static/illustrations/characters/*.webp',
+	{
+		query: {
+			enhanced: true,
```

Comment:
The minimum size it loads by default is 512x512, which is a bit too big for our 40x40 avatars. In order to make smaller sizes, you have to add them here in query, something like:
```suggestion
			enhanced: true,
			w: '1024;512;256;128;64'
```